scripts/deployContractsRollup.ts

```diff
@@ -0,0 +1,49 @@
+import { ethers,run } from 'hardhat';
+import {
+  abi as UpgradeExecutorABI,
+  bytecode as UpgradeExecutorBytecode,
+} from '@offchainlabs/upgrade-executor/build/contracts/src/UpgradeExecutor.sol/UpgradeExecutor.json'
+export async function verifyContract(
+  contractAddress: string,
+  constructorArguments: any
+) {
+  await run("verify:verify", {
+    address: contractAddress,
+    constructorArguments: constructorArguments,
+  });
+}
+async function main() {
+  const ExecutorContract = "0xda8636b0d4f31e87d6e7abe1eaa7a17caa9bc487";
```

Comment:
@RohitAudit what is the function of Executor contract ?

Reply:
executor contract is used to update any params or contract implementation in the nitro contracts